permission_handler/example/android/app/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
+	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="output" path="bin/default"/>
+</classpath>
```

Comment:
This file doesn't belong to source control, please remove it from the PR and add it to .gitignore file (instruction can be found [here](https://stackoverflow.com/questions/18955149/ignore-classpath-and-project-from-git))